pkg/cli/prompt/text/text.go

```diff
@@ -74,7 +77,12 @@ func NewTextModel(prompt string, options TextModelOptions) Model {
 	// so it will be blocked. This means you can't type `prod-aws` which is a valid name.
 	ti := textinput.New()
 	ti.Focus()
-	ti.Width = 40
+	width, err := getTerminalSize()
+	if err != nil {
+		return Model{}, err
+	}
+
+	ti.Width = width
```

Comment:
Do you know what the behavior is if we don't set the width? https://github.com/charmbracelet/bubbles/blob/master/textinput/textinput.go#L118

Does it autoscale?

Reply:
I agree that hardcoding is not the right thing.

Reply:
tested it by not setting the width and it automatically sets the width to teminal width and it autoscales as well. I am going to update the changes to remove the setting of width property.

Reply:
Great 👍 simple solutions are good.